CesiumGltf/include/CesiumGltf/PropertyView.h

```diff
@@ -182,6 +225,39 @@ validateArrayPropertyType(const ClassProperty& classProperty) {
   return PropertyViewStatus::Valid;
 }
 
+/**
+ * @brief Validates a \ref ClassProperty representing an array of enum values,
+ * checking for any type mismatches.
+ *
+ * @returns A \ref PropertyViewStatus value representing the error found while
+ * validating, or \ref PropertyViewStatus::Valid if no errors were found.
+ */
+template <typename T>
+PropertyViewStatusType validateArrayPropertyType(
```

Comment:
See comment on `PropertyTextureView`. I think the same consolidation could be applied here, too.